src/libponyrt/options/options.h

```diff
@@ -45,6 +45,11 @@
   "                   be set to have any effect.\n" \
   "  --ponyversion    Print the version of the compiler and exit.\n" \
   "  --ponyhelp       Print the runtime usage options and exit.\n" \
+  "  --ponysystematictestingseed\n" \
+  "                   Use a specific seed value for systematic testing.\n" \
+  "  --ponysystematictestingverbosity\n" \
+  "                   Control verbosity of systematic testing output.\n" \
+  "                   0 is least verbose; greater is more details.\n" \
```

Comment:
It would be nice to know what the highest meaningful value this could be is.

Is it 3? 5? 9? 99? 999?

Reply:
Right now the max value used is `10` (for no particular reason) but i haven't figured out how the levels of details should work and how far they should go (any ideas?). i expect that the levels of detail is something that will eventually get well defined and stabilized over time.

Reply:
I don't have opinions about how high the levels of verbosity should go - whatever you have now (e.g. 10) is probably fine.

I just think that this documentation string should mention that maximum number so that I (as a user seeing this documentation string) don't need to search through the whole codebase to find out what the highest meaningful number is.